.github/workflows/test.yml

```diff
@@ -7,8 +7,29 @@ on:
     - "v*"
 
 jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
Why v1 instead of v3?

Reply:
v2+ doesn't play nicely with the git command to get `main`'s latest commit